### PR TITLE
Update prerequisite on japan folder

### PR DESCRIPTION
**Pull request by atoato88**
This PR updates prerequisites page on Japan event.
Update contents for ease reading on attendee by converting to desu/masu style, mainly.
And update some items(emphasize items, add details etc..).

Related to: #635

### CODE REVIEW

**Comment by atoato88** [comment]
/assign @s-ito-ts 
Could you review this?

File: japan/assets/attendee-prerequisites.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 # コミュニティ行動規範
 
-まずは、CNCF コミュニティ行動規範をご一読ください
+まずは、CNCF コミュニティ行動規範をご一読ください。
 https://github.com/cncf/foundation/blob/master/code-of-conduct-languages/jp.md
```
Comment (s-ito-ts) [inline japan/assets/attendee-prerequisites.md:7]: nit: How about write the URL on a new line?

Comment (shu-mutou) [reply japan/assets/attendee-prerequisites.md:7]: Rather, let's make it as a link. As same as this, do so for several URLs written in following sentences.

Comment (atoato88) [reply japan/assets/attendee-prerequisites.md]: I've updated commented items.

File: japan/assets/attendee-prerequisites.md [hunk 2]
```diff
@@ -26,51 +25,58 @@ GitHub の Primary email address と Linux Foundation ID のメールアドレ
...
 # Git や GitHub の操作
-* 基本的な Git や GitHub の操作には慣れておいてください。
-* 以下も参考にしてください。
-  + https://github.com/kubernetes/community/blob/master/contributors/guide/contributor-cheatsheet/README-ja.md#%E8%B2%A2%E7%8C%AE%E3%81%99%E3%82%8B
+* 基本的な Git や GitHub の操作に慣れておいてください。
+* [Kubernetesコントリビューターチートシートのこちらの節](https://github.com/kubernetes/community/blob/master/contributors/guide/contributor-cheatsheet/README-ja.md#%E8%B2%A2%E7%8C%AE%E3%81%99%E3%82%8B)から、GitHub での開発の概要を把握しておいてください。
+
 
 # リポジトリのフォークとクローン
 * 以下のリポジトリを自分の GitHub アカウントにフォークしておいてください。
```
Comment (atoato88) [inline japan/assets/attendee-prerequisites.md:82]: I think URL style is better than link text style for GitHub repository at line 83.

File: japan/assets/attendee-prerequisites.md (hunk 1 above)
Comment (s-ito-ts) [reply japan/assets/attendee-prerequisites.md]: Thanks for update.
/lgtm

**Pull request merged**